python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -167,6 +170,7 @@ def evaluate(
                 features_to_return=evaluation_features,
                 additional_features=additional_features,
                 max_sequence_size=self.max_sequence_size,
+                monte_carlo_inference_trials=monte_carlo_inference_trials
```

Comment:
@mohazahran same comment i left elsewhere - i don't think you need to do this if you are using MC scorer.